workers/unity/Packages/io.improbable.gdk.mobile/Utility/MobileConnectionParametersInitializer.cs

```diff
@@ -10,11 +10,15 @@ public void Initialize(ConnectionParameters connectionParameters)
             connectionParameters.Network = new NetworkParameters
             {
                 UseExternalIp = true,
-                ConnectionType = NetworkConnectionType.Kcp,
-                Kcp = new KcpNetworkParameters
+                ConnectionType = NetworkConnectionType.ModularKcp,
+                ModularKcp = new ModularKcpNetworkParameters
                 {
-                    // These are the last tested values
-                    Heartbeat = new HeartbeatParameters
+                    DownstreamHeartbeat = new HeartbeatParameters
```

Comment:
what are the defaults?

Reply:
They were the last tested values :trollface: